lib/pushmi_pullyu/cli.rb

```diff
@@ -120,14 +140,18 @@ def setup_log
   end
 
   def run_tick_loop
+    queue = PushmiPullyu::PreservationQueue.new(connection: { host: config.redis_host, port: config.redis_port },
```

Comment:
This queue initilize can move up into the start_server method. But very minor and we can do this later